spec/features/donuts/01_user_can_view_a_list_of_donuts_spec.rb

```diff
@@ -12,6 +12,8 @@
   let!(:donut) { create(:donut) }
 
   scenario "sees a list of donuts and a link for a new donut" do
+    @user = create(:user)
```

Comment:
Why are these tests being edited in a branch for a user sign out feature? Break these things into different branches & PRs in the future